src/common/libjob/job.h

```diff
@@ -74,9 +71,13 @@ int flux_job_submit_get_id (flux_future_t *f, flux_jobid_t *id);
 flux_future_t *flux_job_list (flux_t *h, int max_entries,
                               const char *json_str);
 
-/* Abort a job.
+/* Raise an exception for job.
+ * Severity is 0-7, with severity=0 causing the job to abort.
+ * Note may be NULL or a human readable message.
  */
-flux_future_t *flux_job_cancel (flux_t *h, flux_jobid_t id, int flags);
+flux_future_t *flux_job_raise (flux_t *h, flux_jobid_t id,
+                               const char *type, int severity,
+                               const char *note);
 
```

Comment:
Any reason not to keep a `flux_job_cancel` as an user-friendly alias for `flux_job_raise`? (I imagine an exception of type `cancel` will be over 90% of use cases, and saving users from writing `flux_job_raise (h, id, "cancel", 0, NULL)` will be a mercy)

Could even just be a macro at this point.

Reply:
OK, will do.